torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -3334,7 +3334,7 @@ def install_subgraph_in_output_graph(
                 gb_type="Encountered non user function variable during invoke_subgraph HOP tracing",
                 context=str(fn_vt),
                 explanation="invoke_subgraph does not support non user function variable",
-                hints=graph_break_hints.SUPPORTABLE,
+                hints=[*graph_break_hints.SUPPORTABLE],
```

Comment:
This looks fine, but why do we need this? graph_break_hints.SUPPORTABLE is already a list

Reply:
I will also note that people will sneak in `hints=graph_break_hints.SUPPORTABLE` unless there's a lint rule or something that enforces it to not look like that